.gitignore

```diff
@@ -1,3 +1,3 @@
 /target
 public/
-
+**/.DS_Store
```

Comment:
this is an OS specific file and should be in a global gitignore, not a project gitignore